src-docs/src/views/inline_edit/inline_edit.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+
+import {
+  EuiInlineEdit,
+  EuiSpacer,
+  EuiButtonGroup,
+  EuiTextProps,
+} from '../../../../src';
+
+export default () => {
+  // Display Text Toggles
+  const textSizeButtons = [
+    {
+      id: 'xs',
+      label: 'Extra Small',
```

Comment:
very minor casing nit ([per our own guidelines](https://eui.elastic.co/pr_6598/#/guidelines/writing/guidelines#capitalization))
```suggestion
      label: 'Extra small',
```

Reply:
Same comment/change request for the title example button options as well